src/Adafruit_SPIFlashBase.cpp

```diff
@@ -57,33 +57,49 @@ Adafruit_SPIFlashBase::Adafruit_SPIFlashBase(
   _flash_dev = NULL;
 }
 
-bool Adafruit_SPIFlashBase::begin(SPIFlash_Device_t const *flash_dev) {
+static SPIFlash_Device_t const *findDevice(SPIFlash_Device_t const *device_list,
+                                           int count,
+                                           uint8_t const (&jedec_ids)[3]) {
+  for (uint8_t i = 0; i < count; i++) {
+    const SPIFlash_Device_t *possible_device = &device_list[i];
```

Comment:
this is really my bad, but would you mind rename `possible_device` to simply `dev` to avoid confusion, especially for non-English speaker.